contract_timesheet_invoice_type/models/account_analytic_line.py

```diff
@@ -8,7 +8,7 @@ class AccountAnalyticLine(models.Model):
     _inherit = "account.analytic.line"
 
     contract_line_id = fields.Many2one(
-        related="project_id.contract_line_id",
+        comodel_name="contract.line"
```

Comment:
When is this field populated?

Reply:
It is currently being populated manually, but the idea is to populate it, as the sale order line is today.

I still need to implement some logic, I focused on structuring the data model in this first "sprint".

I will continue with this in the next 7 days, do you intend to work on it during this period?

Reply:
No I don't intend to work on this short term, please go ahead, this is very cool.